src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -45,12 +42,30 @@ export class GithubAppsLogsComponent implements OnInit {
   showContent: 'table' | 'error' | 'empty' | null;
   isExpansionDetailRow = (i: number, row: Object) => row.hasOwnProperty('detailRow');
 
+  constructor(
+    @Inject(MAT_DIALOG_DATA) public matDialogData: string,
+    private lambdaEventsService: LambdaEventsService,
+    private matSnackBar: MatSnackBar
+  ) {
+    this.datePipe = new DatePipe('en');
+    this.mapPipe = new MapFriendlyValuesPipe();
+    const defaultPredicate = this.dataSource.filterPredicate;
+    this.dataSource.filterPredicate = (data, filter) => {
+      const formatted = this.datePipe.transform(data.eventDate, 'medium').toLowerCase();
+      return formatted.indexOf(filter) >= 0 || defaultPredicate(data, filter);
+    };
+    this.dataSource.filterPredicate = (data, filter) => {
```

Comment:
I don't understand; isn't this going to replace the filterPredicate from line 53?

Reply:
Right, didnt realized having multiple filterPredicates would result in the prior one being overridden. Just fixed it.